RF switch subsystem support
Say Y here if you want to have control over RF switches
found on many WiFi and Bluetooth cards.

To compile this driver as a module, choose M here: the
module will be called rfkill.
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:17.038610372 -0400
@@ -2052,10 +2052,7 @@
 # CONFIG_MAC80211_MESSAGE_TRACING is not set
 # CONFIG_MAC80211_DEBUG_MENU is not set
 CONFIG_MAC80211_STA_HASH_MAX_SIZE=0
-CONFIG_RFKILL=m
-CONFIG_RFKILL_LEDS=y
-CONFIG_RFKILL_INPUT=y
-CONFIG_RFKILL_GPIO=m
+# CONFIG_RFKILL is not set
 CONFIG_NET_9P=m
 CONFIG_NET_9P_FD=m
 CONFIG_NET_9P_VIRTIO=m
@@ -3532,7 +3529,6 @@
 CONFIG_USB_NET_CX82310_ETH=m
 CONFIG_USB_NET_KALMIA=m
 CONFIG_USB_NET_QMI_WWAN=m
-CONFIG_USB_HSO=m
 CONFIG_USB_NET_INT51X1=m
 CONFIG_USB_IPHETH=m
 CONFIG_USB_SIERRA_NET=m
@@ -3560,7 +3556,6 @@
 # CONFIG_ATH9K_STATION_STATISTICS is not set
 # CONFIG_ATH9K_DYNACK is not set
 # CONFIG_ATH9K_WOW is not set
-CONFIG_ATH9K_RFKILL=y
 # CONFIG_ATH9K_CHANNEL_CONTEXT is not set
 CONFIG_ATH9K_PCOEM=y
 CONFIG_ATH9K_PCI_NO_EEPROM=m
@@ -5543,7 +5538,6 @@
 #
 # Media capture support
 #
-CONFIG_VIDEO_MEYE=m
 CONFIG_VIDEO_SOLO6X10=m
 # CONFIG_VIDEO_TW5864 is not set
 # CONFIG_VIDEO_TW68 is not set
@@ -8069,7 +8063,6 @@
 CONFIG_DCDBAS=m
 CONFIG_DELL_LAPTOP=m
 # CONFIG_DELL_RBU is not set
-CONFIG_DELL_RBTN=m
 CONFIG_DELL_SMBIOS=m
 CONFIG_DELL_SMBIOS_WMI=y
 CONFIG_DELL_SMBIOS_SMM=y
@@ -8080,7 +8073,6 @@
 CONFIG_DELL_WMI_DESCRIPTOR=m
 CONFIG_DELL_WMI_LED=m
 CONFIG_DELL_WMI_SYSMAN=m
-CONFIG_AMILO_RFKILL=m
 CONFIG_FUJITSU_LAPTOP=m
 CONFIG_FUJITSU_TABLET=m
 CONFIG_GPD_POCKET_FAN=m
@@ -8088,7 +8080,6 @@
 CONFIG_WIRELESS_HOTKEY=m
 CONFIG_HP_WMI=m
 # CONFIG_IBM_RTL is not set
-CONFIG_IDEAPAD_LAPTOP=m
 CONFIG_SENSORS_HDAPS=m
 CONFIG_THINKPAD_ACPI=m
 CONFIG_THINKPAD_ACPI_ALSA_SUPPORT=y
@@ -8128,7 +8119,6 @@
 CONFIG_INTEL_HID_EVENT=m
 CONFIG_INTEL_VBTN=m
 CONFIG_INTEL_INT0002_VGPIO=m
-CONFIG_INTEL_OAKTRAIL=m
 CONFIG_INTEL_BXTWC_PMIC_TMU=m
 CONFIG_INTEL_CHTDC_TI_PWRBTN=m
 CONFIG_INTEL_CHTWC_INT33FE=m
@@ -8140,7 +8130,6 @@
 CONFIG_INTEL_SMARTCONNECT=y
 CONFIG_INTEL_TURBO_MAX_3=y
 CONFIG_INTEL_VSEC=m
-CONFIG_MSI_LAPTOP=m
 CONFIG_MSI_WMI=m
 CONFIG_PCENGINES_APU2=m
 CONFIG_BARCO_P50_GPIO=m
@@ -8151,11 +8140,8 @@
 CONFIG_TOSHIBA_HAPS=m
 CONFIG_TOSHIBA_WMI=m
 # CONFIG_ACPI_CMPC is not set
-CONFIG_COMPAL_LAPTOP=m
 CONFIG_LG_LAPTOP=m
 CONFIG_PANASONIC_LAPTOP=m
-CONFIG_SONY_LAPTOP=m
-CONFIG_SONYPI_COMPAT=y
 CONFIG_SYSTEM76_ACPI=m
 CONFIG_TOPSTAR_LAPTOP=m
 CONFIG_SERIAL_MULTI_INSTANTIATE=m
